lib/Config.php

```diff
@@ -263,6 +263,22 @@ public static function getWebsiteConfigValue($key = null, $default = null, $lang
         return $config;
     }
 
+    private static function getArrayValueSave($keys, $array)
```

Comment:
We should call this method just `getArrayValue()`, thanks!

Reply:
You are right, I changed the name.